src/libraries/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -132,32 +139,31 @@
     <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.Permissions.cs">
       <Link>Common\Interop\Unix\Interop.Permissions.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.ShmOpen.cs">
+      <Link>Common\Interop\Unix\Interop.ShmOpen.cs</Link>
+    </Compile>
     <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.SysConf.cs">
       <Link>Common\Interop\Unix\Interop.SysConf.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.Unlink.cs">
+      <Link>Common\Interop\Unix\Interop.Unlink.cs</Link>
+    </Compile>
     <Compile Include="Microsoft\Win32\SafeMemoryMappedFileHandle.Unix.cs" />
     <Compile Include="Microsoft\Win32\SafeMemoryMappedViewHandle.Unix.cs" />
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFile.Unix.cs" />
     <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedView.Unix.cs" />
-    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.FTruncate.cs">
-      <Link>Common\Interop\Unix\Interop.FTruncate.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.MAdvise.cs">
-      <Link>Common\Interop\Unix\Interop.MAdvise.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.ShmOpen.cs">
-      <Link>Common\Interop\Unix\Interop.ShmOpen.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)Interop\Unix\System.Native\Interop.Unlink.cs">
-      <Link>Common\Interop\Unix\Interop.Unlink.cs</Link>
-    </Compile>
+    <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFileAcl.Unix.cs" />
+    <Compile Include="System\IO\MemoryMappedFiles\MemoryMappedFileSecurity.Unix.cs" />
   </ItemGroup>
   <ItemGroup>
     <Reference Include="System.IO.FileSystem" />
-    <Reference Include="System.Runtime" />
+    <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Runtime.Extensions" />
     <Reference Include="System.Runtime.InteropServices" />
-    <Reference Include="System.Threading" />
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Security.AccessControl" />
```

Comment:
This is again dependency from inbox binary to a binary shipping as package. Are all combinations of this going to work well?

Reply:
I think this is fine. `System.Security.AccessControl` is OOB only for the ref assembly but the implementation is part of the shared framework:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.AccessControl/Directory.Build.props#L5-L6

Given that we have some inbox projects already depending on it:
https://github.com/dotnet/runtime/blob/02ea20ae6dd0450d04215b561eae449ae1ff289e/src/libraries/System.IO.Pipes/src/System.IO.Pipes.csproj#L125

If I remember correctly we made AccessControl that way specifically to be able to reference it from inbox binaries.
cc: @ericstj